ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard-redesigned.component.ts

```diff
@@ -9,12 +9,15 @@ import { AnnouncementsServiceAgent, SessionMementoService } from 'ddp-sdk';
     selector: 'toolkit-dashboard-redesigned',
     template: `
         <main class="main">
+            <section class="section">
+                <ddp-subject-panel></ddp-subject-panel>
+            </section>
             <section class="section dashboard-title-section">
                 <div class="content content_medium content_wide content_dashboard">
                     <h1 class="dashboard-title-section__title" translate>
                         Toolkit.Dashboard.Title
                     </h1>
-                    <p *ngIf="isAdmin && subjectInfoExists" class="invitation-code">
+                    <p *ngIf="invitationId" class="invitation-code">
```

Comment:
We should show invitation code here only for participants, PR for this feature - #181